src/opsinputs/opsinputs_fill_mod.F90

```diff
@@ -1046,18 +1048,21 @@ subroutine opsinputs_fill_fillreal2d_norecords( &
 integer(c_int), intent(in)                          :: Channels(:)
 character(len=*), intent(in)                        :: JediVarName
 character(len=*), intent(in)                        :: JediVarGroup
-type(opsinputs_channeloffset), optional, intent(in) :: OffsetChans
-logical, optional, intent(in)                       :: useActualChans
+logical, optional, intent(in)                       :: compressVarChannels
+integer(integer64), optional, intent(in)            :: sizeVarObs
+integer(c_int), optional, intent(in)                :: varChannels(:)
+
 
 ! Local declarations:
 real(kind=c_double)                             :: VarValue(NumObs)
 real(kind=c_double)                             :: MissingDouble
-character(len=max_varname_with_channel_length)  :: JediVarNamesWithChannels(max(size(Channels), 1))
+character(len=200)  :: JediVarNamesWithChannels(max(size(Channels), 1))
 integer                                         :: iChannel
 integer                                         :: offset
 integer                                         :: numchans
-logical                                         :: localUseActualChans
+integer                        :: offsetsize
 
+!compressVarChannels=.TRUE.
```

Comment:
If this is a comment line is it still needed?

Reply:
Removed